code/__HELPERS/logging/_logging.dm

```diff
@@ -10,6 +10,31 @@
 	msg = "## NOTICE: [msg]"
 	log_world(msg)
 
+#define SET_SERIALIZATION_SEMVER(semver_list, semver) semver_list[type] = semver
+#define CHECK_SERIALIZATION_SEMVER(wanted, actual) (__check_serialization_semver(wanted, actual))
+
+/// Checks if the actual semver is equal or later than the wanted semver
+/// Must be passed as TEXT
```

Comment:
If you don't want people using this, point them to the macro

Reply:
```suggestion
/// Must be passed as TEXT; you're probably looking for CHECK_SERIALIZATION_SEMVER, look right above
```